docs/guides/08-introduction-to-image-files.md

```diff
@@ -61,15 +61,19 @@ symbol: ramp(buckets($complaint,['Car','Bus','Building']),images([car,bus,house]
 color: ramp(buckets($complaint,['Car','Bus','Building']),[purple,orange,blue])
 ```
 
-http://127.0.0.1:8080/examples/editor/index.html#eyJhIjoidGFibGVfMzExX2NvbXBsYWludHMiLCJiIjoiIiwiYyI6ImNhcnRvdmwiLCJkIjoiaHR0cHM6Ly97dXNlcn0uY2FydG8uY29tIiwiZSI6InN5bWJvbDogcmFtcChidWNrZXRzKCRjb21wbGFpbnQsWydDYXInLCdCdXMnLCdCdWlsZGluZyddKSxzcHJpdGVzKFtjYXIsYnVzLGhvdXNlXSkpXG5jb2xvcjogcmFtcChidWNrZXRzKCRjb21wbGFpbnQsWydDYXInLCdCdXMnLCdCdWlsZGluZyddKSxbcHVycGxlLG9yYW5nZSxibHVlXSlcbndpZHRoOiAyNVxuIiwiZiI6eyJsbmciOi03My45NTYwODMwMjg5MjY3NCwibGF0Ijo0MC43MTIwNzI5MDU0NDY3NDZ9LCJnIjoxNi4xMDc2MDA1MDQ4NzQyNjcsImgiOiJWb3lhZ2VyIn0=
+[Live example](http://carto.com/developers/carto-vl/examples/maps/guides/image-files/color-category-image.html)
```

Comment:
The coloring isn't right here (it is being interpolated) so once we get the new color by category logic merged, this will fix itself.